src/components/cc-article-list/cc-article-list.types.d.ts

```diff
@@ -1,4 +1,19 @@
-export interface Article {
+export type ArticleListState = ArticleListStateLoading | ArticleListStateError | ArticleListStateLoaded;
+
+interface ArticleListStateLoading {
+  state: 'loading';
+}
+
+interface ArticleListStateError {
+  state: 'error';
+}
+
+interface ArticleListStateLoaded {
+  state: 'loaded';
+  articles: Array<Article>;
```

Comment:
**suggestion/question:** what do you think about changing `articles` to `data`? 
This would avoid `this.articles.articles` inside the component :eyes: 

It could also be applied to the `cc-invoice` PR.

Reply:
I do agree that this is not very nice.
This is something that occurs very often when designing the new state. 
Maybe you're suggestion should be applied to all states.
We should maybe discuss that with the rest of the team.

Reply:
Let's go for `value` like for other components